templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,12 +13,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Does the user ever see the actual names for each FOV? If not, I don't think this information is that helpful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
They don't, I'll remove this. If the user really wanted to know the MIBI FOV naming standard they can check out the function or TMA notebook documentation.